packages/components/src/components/NewModal/NewModalBackdrop.tsx

```diff
@@ -0,0 +1,61 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+import FocusLock from 'react-focus-lock';
+import { zIndices, spacings } from '@trezor/theme';
+import { createPortal } from 'react-dom';
+
+import { useModalTarget } from './NewModalProvider';
+import type { UIHorizontalAlignment, UIVerticalAlignment } from '../../config/types';
+
+export type ModalAlignment = { x: UIHorizontalAlignment; y: UIVerticalAlignment };
+
+export type NewModalBackdropProps = {
+    onClick?: () => void;
+    children?: ReactNode;
+    alignment?: ModalAlignment;
+    padding?: number;
```

Comment:
What about adding here `SpacingValues`?

Reply:
I actually don't want that. It's for the device switch modal which requires specific padding not defined in the spacings.

Reply:
I see, it makes sense 👍